### DISCUSSION TITLE
Display Issue

### DISCUSSION THREAD

**Discussion by Swing366** [iTwin/itwinjs-core#1683] [Q&A]
iModel Display in one view, some models similar like be cut at the special angle. See the attachment video.
https://user-images.githubusercontent.com/85770243/122862960-9d9da500-d354-11eb-84e7-b5e7882bbba8.mp4
Thanks in advance.

**Comment by pmconne** [comment]
A couple of things could be causing this:
- The view may have a clip volume applied. If you're not sure, try [turning it off](https://github.com/imodeljs/imodeljs/blob/e982a23bf05019d0a24fd65c9e6fec602d7b4305/core/common/src/ViewFlags.ts#L161).
- The iModel's [project extents](https://github.com/imodeljs/imodeljs/blob/e982a23bf05019d0a24fd65c9e6fec602d7b4305/core/common/src/IModel.ts#L333) may be smaller than the range of the geometry being displayed. Project extents are typically computed by the connector(s) that produced the iModel. What connector(s) produced this iModel?

Are you doing anything weird like displaying multiple iModels in the viewport?
